src/components/va-featured-content/va-featured-content.css

```diff
@@ -0,0 +1,12 @@
+@import '../../mixins/accessibility.css';
```

Comment:
I don't think we need this. Right now that file just contains the `sr-only` class, and it doesn't look like it's being used.

Reply:
Thanks. Updated.